std_msgs/Header header
ObjectBearing[] bearing